src/librbd/cache/pwl/ReplicatedWriteLog.h

```diff
@@ -50,6 +50,43 @@ class ReplicatedWriteLog : public AbstractWriteLog<ImageCtxT> {
   using C_WriteSameRequestT = pwl::C_WriteSameRequest<This>;
   using C_CompAndWriteRequestT = pwl::C_CompAndWriteRequest<This>;
 
+  using AbstractWriteLog<ImageCtxT>::m_lock;
```

Comment:
Nit: same comment here -- if these are getting pulled from the protected section of `AbstractWriteLog`, you really can't reduce their visibility in the private section.

Reply:
Done